resources/views/layout.blade.php

```diff
@@ -43,6 +44,7 @@
         </div>
 
         <footer id="footer">
+            This website is created by James Greenleaf. <a href="https://github.com/aMoniker/wpbinio">Contribute to WPBin.io by forking the GitHub repository</a>.
```

Comment:
Looks good, but just two minor changes here:
- href should be `https://github.com/aMoniker/wpbin`
- we can put something like "This website is open-source." rather than my name since it's not just me working on it now!

Reply:
I've made those changes. Oops at the mistake on the address :facepalm: